docs/sql-ref-ansi-compliance.md

```diff
@@ -156,6 +161,9 @@ SELECT * FROM t;
 +---+
 ```
 
+#### Rounding in cast
```

Comment:
is it also the behavior of casting decimal to int?

Reply:
No, casting to int just removes the fraction part. @gengliangwang @srielau Maybe you what SQL standard says about rounding to int?

Reply:
casting to int behavior is "implementation dependent" The standard makes no decision here.